docs/Kube-API.md

```diff
@@ -129,6 +129,16 @@ In case that the Bare Metal Operator is installed, the Baremetal Agent Controlle
 
 See BMAC documentation [here](./baremetal-agent-controller.md).
 
+## Working with mirror registry
+In case you all your images are in mirror registries, then assisted-service pods, discovery ISO and RHCOS must be provisioned with the proper configuration:
```

Comment:
```suggestion
In case all your images are in mirror registries, assisted-service pods, discovery ISO and RHCOS must be provisioned with the proper configuration:
```

Reply:
fixed